packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -40,31 +37,24 @@ const indeterminateProgress = {
  */
 const useRootStyles = makeStyles({
   root: {
-    display: 'grid',
-    rowGap: '8px',
+    display: 'block',
+    //rowGap: '8px',
     ...shorthands.overflow('hidden'),
```

Comment:
Is this necessary? If you're clamping the width of the bar to be between 0% and 100%, then there should never be overflow right?
```suggestion
```

Reply:
This is necessary because without it, the `indeterminate` `Progress` will run off the track:
![image](https://user-images.githubusercontent.com/66456876/193907554-13307727-e92c-4238-afcc-1cc4ba4d0648.png)

Reply:
Ah ok. One nit in that case: can you just set `overflowX: 'hidden'`?